### PR TITLE
add new browsers, apps and devices

### PR DESCRIPTION
**Pull request by mimmi20**
This PR includes changes from PR #5863 and conflicts with the changes of PR #5783.

### CODE REVIEW

**Comment by sgiehl** [comment]
I see what you are trying to do with adding new tests. Could you maybe move adding your new fixtures after the already existing fixtures have been sorted? That way they will simply be added at the end and it's easier to see the actual changes and it won't conflict with other PRs that much.

Also I'm not sure why the surrounding `"` from the version numbers have been removed everywhere.
That actually doesn't hurt for the PHP tests here, but some of the ports to other languages are using the test fixtures here, and they have a problem as they are "detected" as invalid numbers without those surrounding `"`
Guess the script to add those chars didn't work on your system:
https://github.com/matomo-org/device-detector/blob/6bb8507bd06f041b073924368f18f0b7f74b7402/misc/sortTestFixtureFiles.php#L116

**Comment by mimmi20** [comment]
@sgiehl The PR is not finished yet. I work on a windows system, and `sed` doesn't work there. This will be fixed tomorrow. I used the `misc/sortTestFixtureFiles.php` file to rewrite and add the test cases.

**Comment by mimmi20** [comment]
@sgiehl You may review now.

File: Parser/Client/Browser.php [hunk 1]
```diff
@@ -71,15 +73,18 @@ class Browser extends ClientParserAbstract
         'CY' => 'Cyberfox',
         'CS' => 'Cheshire',
         'CU' => 'Cunaguaro',
+        'CV' => 'Chrome Webview',
+        'DA' => 'Dalvik',
```
Comment (Findus23) [inline Parser/Client/Browser.php]: Dalvik isn't a browser, but something android webviews sometimes send as a useragent:
https://stackoverflow.com/questions/23804278/browser-sending-dalvik-as-user-agent

**Review by Findus23 — CHANGES_REQUESTED** [review]
I can't say much about the non-browser changes, but I am not sure if spliting up iOS/mobile browser variants is that useful. After all DeviceDetector is already able to keep them apart via the OS detection.

**Comment by mimmi20** [comment]
I will split the PR.

**Pull request closed without merging**